build-tools/packages/build-cli/src/instructionalPromptWriter.ts

```diff
@@ -44,6 +44,59 @@ interface Section {
     cmd?: string;
 }
 
+/**
+ * Links to ADO pipeline for all release group
+ */
+
+export enum ADOPipelineLinks {
```

Comment:
Suggest making this a string-map. E.g.

```ts
export const ADOPipelineLinks = new Map<ReleasePackage | ReleaseGroup, string>([
  [ReleaseGroups.Client, "URL"],
  ["@fluidframework/common-utils", "URL"],
  ...
]);
```

This way the data is indexed by ReleaseGroup or package, which makes looking up the links a lot less code. You may need to play around with the typing -- I don't know if you can use a union type as a key.